### ISSUE TITLE
Test Prototype: Cancer Types Landing Page (A-Z List)

### ISSUE THREAD

**Issue by laurelthrash** [NCIOCPL/ncids#570]
# Test Prototype: Cancer Types Landing Page (A-Z List)
*Summary: Determined that we would create a prototype for the Cancer Types Landing Page (A-Z List) and test the prototype to verify our design. This ticket outlines the testing strategy for the prototype.*

### Links to related tickets:
- Discovery Ticket: #384 
- Direction Ticket: #491 
- Initial Prototypes Ticket: #525  
- Testing Prototypes Ticket (this ticket): #570 
- Solution Ticket: #504 
- Build Ticket(s): TBD

### Final A-Z Comp 

Approved by federal team on 1/26/23: https://publicis.invisionapp.com/console/share/GW5Z2TJMUHB

### Sub Tasks

**Sprint 38 [SP:3]**
- [x] Establish what we want to learn from this research.
- [x] Confirm/establish audience for testing.
- [x] Create screener. 
- [x] Federal approval of the screener via email. 

**Sprint 40: [SP: 5]**
- [x] Create guide/script to walk through with participants. *Shared for internal review on 09/29. Completed internal review on 10/4, with summary in the comments. *
- [x] Kick-off recruiting (internal and external) for qualified participants to take part in scheduled moderated user sessions. (Determined on 10/11 at mid-sprint that we will divide the participants into 5 and 4 in order to test both desktop and mobile)

**Sprint 41 [SP: 5]**
- [x] Finalize testing schedule
- [x] Conduct the testing with participants (week of 10/24). (Note: this includes testing for desktop and mobile)

**Sprint 42 [SP: 5]**
- [x] Generate analysis/report with findings (desktop and mobile).
- [x] Determine next steps with sharing with the internal team/client.

**Sprint 43 [SP: N/A]**
- [x] Identify comp updates based on findings.  
     - [x] Conduct initial brainstorm **Scheduled for Thurs 11/29 @ 1pm**

**Sprint 44 [SP: 3]**
- [x] Update to the styles (fonts, colors, hyperlink styles) to match R2 for desktop and mobile (one aspect of feedback from brainstorm on 11/29)

**DS Sprint 7 [SP: 5]**
- [x] Complete remainder of comp updates based on 11/29 brainstorm @blilianyu @monika-jaeger 
- [x] Verify comps use USWDS aligned tokens (specifically, spacing and shadow tokens)
- [x] Review comp updates internally **(scheduled for Wed. 12/21 @ 11:00am)**
- [x] Make updates to comps based on 12/21 conversation
- [x] Share comps internally via Slack and obtain final review/approval [completed in DS Sprint 8]

**DS Sprint 9 [SP: N/A]**
- [x] Schedule time to review findings and comps with federal team [PM to schedule] (goal is to be completed in DS Sprint 8)
- [x] Conduct review with federal time

**DS Sprint 10 [SP: TBD]**
- [x] Share with Federal Team via Teams for final approval
- [x] Obtain final federal approval

### Timeline
*Identify start and end times for solution work.*

**Comment by laurelthrash** [comment]
Seeking internal review of the screener. Liz shared via Slack.

**Comment by laurelthrash** [comment]
Capture SP3 in sprint 38 and move SP5 to sprint 39.

**Comment by laurelthrash** [comment]
Discussed internally on 10/4. Next steps and to-dos are below:

- [ ] Expand "B" accordion (in addition to "E") so that the existing script is seamless. Due by Friday 10/07 @acsnedegar 
- [ ] Request that work "contract" in moderator guide be replaced with a word such as "Close" or "Collapse" for readability @lizsiciliano 
- [ ] Determine status of sub-task "Recruit (internal and external) qualified participants..." before marking as complete @lizsiciliano 

Summary:
- Move this ticket to Sprint 40 in order to fully account for above tasks.

**Comment by laurelthrash** [comment]
@acsnedegar updated the prototypes so that the 'B' expands and collapses. Updated prototype links are below:
- [desktop](https://publicis.invisionapp.com/console/share/N6STFZAMCD8)
- [mobile](https://publicis.invisionapp.com/console/share/S4PGJEYH3Z8)

**Comment by laurelthrash** [comment]
Completed sub-tasks outlined in Sprint 40 (SP5). Will carry over remaining tasks to Sprint 41. Cc: @lizsiciliano

**Comment by laurelthrash** [comment]
(Comment in-progress)

Discussed internally on 11/10. Liz walked internal team through the findings and analysis. Next steps defined in the description. 

Notes:
- Consider language change in the combo box
- Consider revisiting links for "Related" section

**Comment by laurelthrash** [comment]
(In Progress) Internal discussion held on 11/29:

Actions to Consider 
- [ ] Remove "Related Information" heading in favor of intro text (mentioned below) [Possible language: "Point of Diagnosis,"  "...and other types of cancer" 
- [ ] Add a description that connects the items in "Related Information" with what information users will find on the page (in response to users feeling like these are random links) - specifically, add description/information to complement "Metastatic Cancer" and "Recurrent Cancer"
- [ ] Modify so intro text appears before "Common Cancer Types" to speak to what users will see in "Common Cancer Types" [Below "Find a cancer type" and above "Common Cancer Types"]
- [ ] Need tp update comps with all styles from MVP1 styles

Notes
- Consider converting "Related Information" into a section with more text
- Add text to describe what they will find on the page in relation to type (e.g., adolescent, adult) 
- Consider removing "Related Information"
- Would not consider "floating" since we recently removed the delighters

**Comment by monika-jaeger** [comment]
Comps with updated designs per 11/29 discussion: [InVision Link](https://publicis.invisionapp.com/console/share/QKTBC3HGFRD).

**Comment by laurelthrash** [comment]
(In Progress) 
Internal discussion on 12/21:

Actions:
- [ ] Reform "Point of Diagnosis" text box

Notes:
- Try to balance content up top and accessibility to get to the letters

**Comment by adrianacastaneda** [comment]
@monika-jaeger Lindsay and I've agreed on the heading "Advanced and Ongoing Diagnosis" 

Along with this heading we'd like to add the following links to the box:
- [Advanced Cancer](https://www.cancer.gov/about-cancer/advanced-cancer)
- [Managing Cancer Care](https://www.cancer.gov/about-cancer/managing-care)


cc: @laurelthrash

**Comment by monika-jaeger** [comment]
Updated comps with text for the 'Advanced and Ongoing Diagnosis' & removed subtext. [InVision Link](https://publicis.invisionapp.com/console/share/2G5C6QHF387).

cc @adrianacastaneda @lburack @laurelthrash

**Comment by monika-jaeger** [comment]
[Final Comps for Federal Review](https://publicis.invisionapp.com/console/share/JU34QGFTPBZ)

**Comment by laurelthrash** [comment]
Review meeting scheduled for 1/24/23.

**Comment by laurelthrash** [comment]
[Comment in progress]

Changes to comps

- [ ] Change "Advanced and Ongoing Diagnosis" label for representation on comps

Notes/Discussion

- Monika showed version that was tested in addition to updated comp

**Comment by laurelthrash** [comment]
A-Z comp approved by federal team on 1/26/23 via Teams. Made following adjustments from the original design that were approved: 

1. Increased the size of the letters of the alphabet as well as updated the +/- to align with USWDS
2. Changed the color of the accordion to align with USWDS requirements. Since this is the same color as the page background color we changed the background to white. The accordions now align with the [USWDS implementation of the accordion (bordered accordion)](https://designsystem.digital.gov/components/accordion/).

Closing this ticket and following up on next steps.
